releasenotes/notes/apm-allow-configuration-of-security-context-for-auto-instrumentation-init-containers-28ab0a87e1920831.yaml

```diff
@@ -0,0 +1,18 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    APM: In order to allow for auto instrumentation to work in Kubernetes
+    clusters that enforce a ``Restricted`` `Pod Security Standard <https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted>`_,
+    which require all containers to explicitly set a ``securityContext``,
+    an option to configure a `securityContext` to be used for all ``initContainers``
```

Comment:
```suggestion
    adds an option to configure a `securityContext` to be used for all ``initContainers``
```

Reply:
@hestonhoffman Now that I re-read this I see that the sentence didn't make any grammatical sense, thanks for catching it 😅 
I originally intended the sentence to end with "...  have been added." instead, which I've added now - hope that's still fine? 😄

Reply:
Let's use "has been added" instead :)

Reply:
Indeed 🙈 😅 
Fixed 😉